### ISSUE TITLE
Make Python-magic optional

### ISSUE THREAD

**Issue by scj643** [randomchars/pushbullet.py#53]
Python-magic is only needed if your pushing files so it shouldn't be a hardcoded dependency. Also some platforms like Pythonista (iOS app for coding Python) and windows 64 bit don't have an easy of of getting and using Python-magic.

**Comment by r-darwish** [comment]
951469c6b24643f5b8b9abf106a13646ae9ba2ba was made to address this. Are you still experiencing issues with recent versions?

**Comment by scj643** [comment]
Using pip to install doesn't work because pip requires it.

**Comment by r-darwish** [comment]
I checked Python 3.5.0 (both 32 and 64 bit) in Windows 10 where there's no libmagic. python-magic is installed successfully (although it cannot be imported) and pushbullet.py seem to work as expected. It may be possible that Pythonista has a specific issue with installing python-magic. If that's the case then adding an if statement that prevents adding python-magic to the dependency list if running under Pythonista might do the trick. Unfortunately I have no experience with Pythonista nor do I have an iOS device to test it with. You're welcome to submit a pull request with the change I suggested.
